src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -49,13 +49,17 @@ public Shooter(ShooterMap shooterMap) {
 
     public Command setSpeed(Speeds speed) {
         PersistenceCheck speedPersistenceCheck = new PersistenceCheck(5, () -> {
-            return Math.abs(getData().leftWheels.velocity) > speed.getLeftSpeed(getMap().splitSpeeds)
-                    && Math.abs(getData().rightWheels.velocity) > speed.getRightSpeed(getMap().splitSpeeds);
+            return Math.abs(getData().leftWheels.velocity) > speed.getLeftSpeed(getMap().splitSpeeds) - 50
```

Comment:
We should turn these numbers into constants `private final double BLAH_BLAH_BLAH`